package.json

```diff
@@ -201,11 +202,11 @@
     "@typescript-eslint/parser": "^6.11.0",
     "@typescript-eslint/types": "^6.11.0",
     "@yarnpkg/sdks": "^3.0.0",
-    "ag-grid-angular": "~31.0.0",
-    "ag-grid-community": "~31.0.0",
+    "ag-grid-angular": "~31.0.3",
```

Comment:
is it still mandatory at workspace level?

Reply:
seems not